src/cljs/webchange/editor_v2/translator/text/views.cljs

```diff
@@ -0,0 +1,142 @@
+(ns webchange.editor-v2.translator.text.views
+  (:require
+    [cljs-react-material-ui.reagent :as ui]
+    [re-frame.core :as re-frame]
+    [reagent.core :as r]
+    [webchange.editor-v2.components.confirm-dialog.views :refer [confirm-dialog]]
+    [webchange.editor-v2.translator.translator-form.state.form :as translator-form]
+    [webchange.editor-v2.translator.translator-form.state.scene :as translator-form.scene]
+    [webchange.editor-v2.translator.text.core :refer [parts->chunks chunks->parts]]))
+
+(def modal-state-path [:editor-v2 :translator :text :configuration-modal-state])
```

Comment:
For consistency in work with app state should we separate subscriptions and events registrations from views into folder 'state'? 

In this case we also should rely on 'webchange.editor-v2.translator.state.db' (as we do in webchange.editor-v2.translator.translator-form.state.db for example) during state path generation to make state node renaming (possible) process easier